docs/servo_power_monitor/prototype/serialport/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>serial_connection</title>
+    <style>
+        body {
+            margin: 0;
+            padding-bottom: 3rem;
+            font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Comment:
Probably using Mono space font is nice (at least for the console output).

Following pages may help you:
- https://fonts.google.com/?classification=Monospace
- https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#some_common_font_families

Reply:
even just using
```
font-family: monospace;
```
will give us a better appearance ;)

Reply:
font-family: monospace is the standard. Don't pull unnecessary fonts for the time being.